08-reactivity/02-scroll-after-update/components/MiniMessenger.vue

```diff
@@ -44,7 +46,21 @@ export default {
       this.newMessage = '';
     },
   },
-};
+
+  watch: {
+    async ['messages.length']() {
+      const list = this.$refs.items[0]?.parentElement;
```

Comment:
Previous problem - if we need a container of messages, we should add a new `ref` in the template.